src/main/java/com/projetoGerenciamentoCurso/gerenciamentoCurso/controller/ProfessorController.java

```diff
@@ -14,63 +14,58 @@
 import com.projetoGerenciamentoCurso.gerenciamentoCurso.dto.ProfessorDto;
 import com.projetoGerenciamentoCurso.gerenciamentoCurso.models.Professor;
 import com.projetoGerenciamentoCurso.gerenciamentoCurso.service.ProfessorService;
-import com.projetoGerenciamentoCurso.gerenciamentoCurso.service.repositoryServices.ProfessorRepositoryService;
 
 
 @RestController
 @RequestMapping(path = ProfessorController.PATH)
 public class ProfessorController {
 
-	@Autowired
-	private ProfessorRepositoryService professorRepositoryService;
+	
 	
 	@Autowired
 	private ProfessorService professorService;
 	
 	public static final String PATH = "/professor";
 	
-
 	
 	@PostMapping
 	@Transactional
 	public ResponseEntity<Professor> salvaProfessor(@RequestBody ProfessorDto professor) {
-		Professor professorModel = professorRepositoryService.criarModelProfessor(professor);
-		professorRepositoryService.repositoryCriarProfessor(professorModel);
-		professorService.criarProfessor(professor);
+		Professor professorModel = professorService.criarProfessor(professor);
+		
 		return ResponseEntity.ok(professorModel);
 	}
 	
 	@PutMapping
 	@Transactional
 	public ResponseEntity<Professor> atualizaProfessor(@RequestBody ProfessorDto professor) {
-		Professor atualizaProfessor = professorRepositoryService.repositoryEncontrarProfessor(professor.getIdPessoa());
-		if(atualizaProfessor == null) {
+		Professor atualizaProfessor = professorService.atuzalizarProfessor(professor);

```

Comment:
Corrigir o nome do método do Service
```suggestion
		Professor atualizaProfessor = professorService.atualizarProfessor(professor);
```

Reply:
feito